batch_training/model_from_scratch/README.md

```diff
@@ -0,0 +1,55 @@
+# Batch training
+
+This readme describes how to perform batch training on a logistic regression model created from scratch 
+
+## How to perform batch training with Cape
+
+### Prepare deployment folder with dependencies
+Create a depoyment folder called `app`:
+```
+mkdir app
+```
+Save app.py into the folder:
+```
+cp app.py ./app
+```
+Save untrained model into the folder:
+```
+cp params.json ./app
+```
+Install necessary libraries listed in `requirements.txt`
+```
+sudo docker run -v `pwd`:/build -w /build --rm -it python:3.9-slim-bullseye pip install -r requirements.txt --target ./app/
```

Comment:
Loving how thorough this readme is so far! It could be cool to make this command into a utility at some point (build it into  the cape cli or something) if it's always the same parameters

Reply:
good idea